ext/include/opentelemetry/ext/http/client/http_client.h

```diff
@@ -227,8 +227,11 @@ class HttpClient
 {
 public:
   virtual std::shared_ptr<Session> CreateSession(nostd::string_view host,
-                                                 uint16_t port = 80) noexcept = 0;
-  virtual bool CancelAllSessions() noexcept                                   = 0;
+                                                 uint16_t port) noexcept = 0;
```

Comment:
Do we still need the port parameter here, if the URL itself can contain scheme and port?
Should we rename the `host` parameter to `url` or `uri` ?